Be-Secure/BeSLighthouse
# Added workflow for release

## Testing

1. Clone repo
2. Tag the code using `git tag -a <version> -m "<msg>"
3. Push tag using `git push origin <version>
4. Go to repo and check actions for log.
5. Check for the latest release.
6. Check the gh-pages deployment for latest deployment of the website.
